test/wrapper_test.rb

```diff
@@ -3204,4 +3188,72 @@ def test_empty_result_when_no_mission
 
     assert_equal 2, result[:routes].size
   end
+
+  def test_assert_inapplicable_for_vroom_if_vehicle_distance
```

Comment:
we could actually define the timewindow to the maximum distance as we are sure we have no timewindow currently.

But for now, this test is usefull ;)

Reply:
Does that mean that if there is no time dimension we can use TW's to implement max distance?

Reply:
Yes !